.github/workflows/analyze.yml

```diff
@@ -40,11 +40,11 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: true
-      - name: Set up Java 11
+      - name: Set up Java 17
         uses: actions/setup-java@v2
         with:
-          java-version: 11
-          distribution: 'adopt'
+          java-version: 17
+          distribution: 'temurin'
```

Comment:
Maybe I should close #366 ? Feel free to close it if so.

Reply:
Ok, I cherry picked your commit so we keep a trace of your contribution. Thanks a lot, I wasn't aware of the ability to use variables in workflows.

Reply:
TBH, I just copy/paste the example from the action/java repo :sweat_smile: